cropper/app/lib/CropStore.scala

```diff
@@ -56,15 +63,19 @@ class CropStore(config: CropperConfig) extends S3ImageStorage(config) {
           val updatedCrop = for {
             // Note: if any is missing, the entry won't be registered
             source <- userMetadata.get("source")
-            x      <- userMetadata.get("bounds_x").map(_.toInt)
-            y      <- userMetadata.get("bounds_y").map(_.toInt)
-            w      <- userMetadata.get("bounds_w").map(_.toInt)
-            h      <- userMetadata.get("bounds_h").map(_.toInt)
+
+            // we've moved to kebab-case as localstack doesn't like `_`
+            // fallback to reading old values for older crops
+            // see https://github.com/localstack/localstack/issues/459
```

Comment:
They claim this is 'kludged'. Possibly not merged. Makes me sad that we are working around an issue in a localstack but 🤷